gateway/gateway.go

```diff
@@ -1105,6 +1104,26 @@ func (gateway *HandleT) StartWebHandler() {
 	gateway.logger.Fatal(srv.ListenAndServe())
 }
 
+//AdminHandler for Admin Operations
+func (gateway *HandleT) StartAdminHandler() {
+	gateway.logger.Infof("Starting AdminHandler in %d", adminWebPort)
+	srvMux := mux.NewRouter()
+	srvMux.Use(headerMiddleware)
+	srvMux.HandleFunc("/v1/clear", gateway.stat(gateway.ClearHandler)).Methods("POST")
+	srvMux.HandleFunc("/v1/pending-events", gateway.stat(gateway.pendingEventsHandler)).Methods("POST")
+
+	srv := &http.Server{
+		Addr:              ":" + strconv.Itoa(adminWebPort),
+		Handler:           bugsnag.Handler(srvMux),
+		ReadTimeout:       config.GetDuration("ReadTimeOutInSec", 0*time.Second),
```

Comment:
We are using same configuration for timeouts in both servers? I see you have changed default value for WriteTimeout. but overriding the configuration in config.yaml will change both servers as they are using same config key right?